Editor/Gui/Windows/ScreenManager.cs

```diff
@@ -0,0 +1,441 @@
+            }
+            catch
+            {
+                // Alternative method 2 - Direct display properties
+                try
+                {
+                    Process.Start("desk.cpl");
+                }
+                catch (Exception fallbackEx)
+                {
+                    // Log the error or show a message to the user
+                    Debug.WriteLine($"Failed to open display settings: {ex.Message}");
+                    Debug.WriteLine($"Fallback also failed: {fallbackEx.Message}");
+                }
+            }
+        }
+    }
+
+    private static bool IsSpanningAreaLargerThanScreens(Screen[] screens, Vector4 spanningArea)
+    {
+        if (spanningArea.Z == 0 || spanningArea.W == 0)
+            return false;
+
+        // Get all screens that are currently in the spanning area
+        var screensInSpanning = screens.Where(s => IsScreenInSpanningArea(s, spanningArea)).ToArray();
+
+        if (screensInSpanning.Length == 0)
+            return false;
+
+        // Calculate the total area of all screens in the spanning area
+        var totalScreenArea = screensInSpanning.Sum(s => s.Bounds.Width * s.Bounds.Height);
+
+        // Calculate the spanning area
+        var spanningAreaTotal = spanningArea.Z * spanningArea.W;
+
+        // If spanning area is larger than the sum of screen areas, there are gaps
+        return spanningAreaTotal > totalScreenArea;
+    }
+
+    private static void RefreshScreenCache()
+    {
+        var newScreens = Screen.AllScreens;
+
+        // Trigger layout rebuild if number or names differ
+        var screensChanged =
+            _cachedScreens.Length != newScreens.Length ||
+            !_cachedScreens.Select(s => s.DeviceName).SequenceEqual(newScreens.Select(s => s.DeviceName));
+
+        // Trigger rebuild if bounds changed (e.g., moved monitors or changed resolution)
+        if (!screensChanged)
+        {
+            for (var i = 0; i < newScreens.Length; i++)
+            {
+                if (!_cachedScreens[i].Bounds.Equals(newScreens[i].Bounds))
+                {
+                    screensChanged = true;
+                    break;
+                }
+            }
+        }
+
+        if (screensChanged)
+        {
+            _cachedScreens = newScreens;
+            _cachedOverallBounds = GetOverallScreenBounds(newScreens);
+            _layoutDirty = true;
+        }
+
+        // Always refresh spanning area, since user may adjust it via settings
+        _cachedSpanning = UserSettings.Config.OutputArea;
+
+        // Refresh selected spanning screens
+        _screensInSpanning.Clear();
+        foreach (var s in _cachedScreens)
+            if (IsScreenInSpanningArea(s, _cachedSpanning))
+                _screensInSpanning.Add(s);
+    }
+
+
+    // -------------------------------------------
+    // Helper: Compute scale to fit screens in UI
+    // -------------------------------------------
```

Comment:
```suggestion
    /** Helper: Compute scale to fit screens in UI  **/
```

Please use consistent comment style. Using the .net style will make documentation available on hover.